README.rst

```diff
@@ -29,6 +29,9 @@ Requirements
 Calling ``source ./tools/bootstrap.sh`` will put you in a
 virtualenv with all dependencies installed (except graphviz).
 
+Afterwards, you may need to set ``PYTHONPATH`` to
+``_venv/lib/python*/site-packages/``, where ``*`` is the
+version number of your python installation.
```

Comment:
Something isn't right here, that should be handled by the virtualenv

Reply:
If I try to run ./tools/build.sh, it says it cannot find "pgi". _venv/bin/activate modifies PATH so there is a private copy of python, etc but still cannot find the libs. By the way python itself is small - there is libpython elsewhere.

I am not familiar with virtualenv, and documentation is poor. Is there any way of checkin what it is supposed to be doing and what it is not?

Reply:
I did a bit of digging, and found something curious. If I start python interactively, it can find pgi:

```
(_venv) [Hin-Tak@localhost pgi-docgen]$ python
Python 3.6.6 (default, Jul 19 2018, 14:25:17) 
[GCC 8.1.1 20180712 (Red Hat 8.1.1-5)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pgi
>>> 
```

and if I just run the content of `pgidocgen.py` it works:

```
$ python
Python 3.6.6 (default, Jul 19 2018, 14:25:17) 
[GCC 8.1.1 20180712 (Red Hat 8.1.1-5)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> from pgidocgen.main import main
>>> import pgi
>>> main(["pgidocgen.py","create","_docs", "HarfBuzz-0.0"])
```
but somehow running pgidocgen.py via `tools/build.sh` does not work.